<?xml version="1.0" encoding="UTF-8" ?>
<launch>

    <node pkg="inverted_pendulum_sim" type="inverted_pendulum_sim_node.py" name="InvertedPendulum" output="screen"
          required="true"/>
    <node pkg="controller_pkg" type="task1.py" name = "task1" output="screen" />
    <node pkg="controller_pkg" type="task3.py" name = "task3" output="screen" />
    <node name="theta" pkg="rqt_plot" type="rqt_plot" output="screen" args="/inverted_pendulum/current_state/curr_theta" />
   
</launch>